src/components/_button.scss

```
@@ -216,6 +216,7 @@ $sb-btn-link-padding: sb-rem-spacing('xs');
 // .sb-btn--danger - Danger button
 // .sb-btn--warning - Warning button
 // .sb-btn--success - Success button
+// .sb-btn--interactive - Success button
```

```suggestion
// .sb-btn--interactive - Interactive button
```

maybe primary outline something like that, all that buttons are interactive right!

---

When I talked to @Maruceau that was the name it was given.
I agree we might need to discuss the naming and decide. But it's defintely not primary outline. It wlll currently by applied to some subtle buttons